viewer/packages/pointclouds/src/annotationTypes.ts

```diff
@@ -34,7 +35,22 @@ export class BoxPrimitive {
 
 export type Geometry = CylinderPrimitive | BoxPrimitive;
 
-export type BoundingVolume = {
+export type CdfPointCloudObjectAnnotation = {
   annotationId: number;
+  assetId?: number;
   region: Geometry[];
 };
+
+export type AnnotationMetadata = {
+  annotationId: number;
+  assetId?: number;
+};
+
+export type PointCloudObjectAnnotation = AnnotationMetadata & {
+  stylableObject: StylableObject;
+};
+
+export type PointCloudObjectAnnotationsWithIndexMap = {
```

Comment:
Can this be a proper class, e.g. `PointCloudObjectProvider` or something, with relevant functionality?